compose/travis/Dockerfile

```diff
@@ -12,11 +12,8 @@ ENV PYTHONUNBUFFERED 0
 
 ENV BIGCHAINDB_DATABASE_PORT 27017
 ENV BIGCHAINDB_DATABASE_BACKEND $backend
-ENV BIGCHAINDB_SERVER_BIND 0.0.0.0:9984
```

Comment:
I don't understand why these Dockerfile ENV variables are being removed in this PR. Same for the corresponding settings removed in `docker-compose.travis.yml`.

Reply:
These variables are not needed by the travis configuration, but they were failing the tests, I enabled because default values for BIGCHAINDB_SERVER_BIND is `localhost:9984`, BIGCHAINDB_WSSERVER_HOST is `localhost` and BIGCHAINDB_WSSERVER_ADVERTISED_HOST is `localhost`. Which made some tests fail that assert the default values. Either I needed to change here since this is not being used OR change the default values in the BigchainDB config. But if we set it to `0.0.0.0`, some use cases will need to be updated as well.

Reply:
Okay maybe leave them removed then. I just didn't understand why removal of some CLI subcommands would require changing `compose/travis/Dockerfile` in this way, but now I do.

Reply:
I doubt very strongly that the env vars should be changed or removed. Running the tests in a dev environment with docker-compose fails for some tests now. The tests, or config mechanism needs to be fixed not the docker files. 

If the same modifications are applied to the docker-compose file used for development the following error occurs:

```bash
bdb_1          | Process ws:                                                                             
bdb_1          | Traceback (most recent call last):                                                      
bdb_1          |   File "/usr/local/lib/python3.6/asyncio/base_events.py", line 1044, in create_server   
bdb_1          |     sock.bind(sa)                             
bdb_1          | OSError: [Errno 99] Cannot assign requested address                                                           
bdb_1          |                                                                                         
bdb_1          | During handling of the above exception, another exception occurred:                                           
bdb_1          |                                                                                         
bdb_1          | Traceback (most recent call last):            
bdb_1          |   File "/usr/local/lib/python3.6/multiprocessing/process.py", line 258, in _bootstrap   
bdb_1          |     self.run()                                                                          
bdb_1          |   File "/usr/src/app/bigchaindb/utils.py", line 38, in run                                                    
bdb_1          |     super().run()                                                                       
bdb_1          |   File "/usr/local/lib/python3.6/multiprocessing/process.py", line 93, in run                                 
bdb_1          |     self._target(*self._args, **self._kwargs)                                           
bdb_1          |   File "/usr/src/app/bigchaindb/web/websocket_server.py", line 173, in start                                  
bdb_1          |     port=config['wsserver']['port'])          
bdb_1          |   File "/usr/local/lib/python3.6/site-packages/aiohttp/web.py", line 457, in run_app                          
bdb_1          |     asyncio.gather(*server_creations, loop=loop)                                                              
bdb_1          |   File "/usr/local/lib/python3.6/asyncio/base_events.py", line 467, in run_until_complete                     
bdb_1          |     return future.result()                    
bdb_1          |   File "/usr/local/lib/python3.6/asyncio/base_events.py", line 1048, in create_server                         
bdb_1          |     % (sa, err.strerror.lower()))
bdb_1          | OSError: [Errno 99] error while attempting to bind on address ('::1', 9985, 0, 0): cannot assign requested add
ress
```

Some key parts of the application (seems to be `bigchaindb/web/websocket_server.py` in this case) are not tested, and that is most probably why removing the env vars for travis worked, but this can hardly be a path forward.